src/stores/MessageStore.tsx

```diff
@@ -44,20 +44,22 @@ export class MessageStore {
         this.fetchMessages(filterAddress);
     }
     public sendMessage = flow(function* (this: MessageStore, reciver: Contact, messageText: string) {
-        this.state = MessageStoreState.sending
-        const msg: Message = {
+        if(messageText.trim().length > 0) {
```

Comment:
why do you double check if message length is 0?

Reply:
Because the check in MessageStore is not related to chat application. That means, if we want to extend the app and add some sort of other feature that uses the MessageStore to send a message, it checks it there.
The other one in Sender is to prevent any empty string being sent via the Sender component.
Perhaps I might have overthought it.